code/modules/mob/living/carbon/life.dm

```diff
@@ -274,15 +274,17 @@
 
 		return 1
 
+
+
 /mob/living/carbon/proc/CheckStamina()
-	if(staminaloss)
-		var/total_health = (health - staminaloss)
-		if(total_health <= config.health_threshold_crit && !stat)
-			src << "<span class='notice'>You're too exhausted to keep going...</span>"
-			Weaken(5)
-			setStaminaLoss(health - 2)
-			return
-		setStaminaLoss(max((staminaloss - 2), 0))
+    if(staminaloss)
```

Comment:
I just checked it and there doesn't seem to be any issues with indentation, I'm pretty sure dreammaker would have spat out an error about inconsistent indentation if it was fucked up here.